generators/entity-server/index.js

```diff
@@ -71,34 +73,81 @@ module.exports = class extends BaseBlueprintGenerator {
         return this._writing();
     }
 
+    _preparing() {
+        return {
+            processJavaEntityImports() {
+                this.importJsonIgnore = false;
+                this.importJsonIgnoreProperties = false;
+                this.importSet = false;
+                this.uniqueEnums = {};
+
+                this.importApiModelProperty = Object.values(this.relationships).filter(v => typeof v.javadoc != 'undefined').length > 0;
+                if (!this.importApiModelProperty) {
+                    this.importApiModelProperty = Object.values(this.fields).filter(v => typeof v.javadoc != 'undefined').length > 0;
+                }
+
+                Object.values(this.relationships).forEach(v => {
```

Comment:
Great.
Rename `v` to `relationship`
```suggestion
                this.relationships.forEach(v => {
```

Reply:
Done